### ISSUE TITLE
Convert_tf_checkpoint_to_pytorch for bert-joint-baseline

### ISSUE THREAD

**Issue by raheja** [huggingface/pytorch-pretrained-BERT#449]
Hello,

I know that the format of Squad and Google NQ is different, but is there a way to convert the bert joint model for Natural Questions (https://github.com/google-research/language/tree/master/language/question_answering/bert_joint) to pytorch? I get this error
'BertForPreTraining' object has no attribute 'answer_type_output_bias'

**Comment by Single430** [comment]
I also encountered a similar problem AttributeError: 'BertForPreTraining' object has no attribute 'crf_loss'

**Comment by thomwolf** [comment]
Hi, from my reading of the [Natural Questions model](https://github.com/google-research/language/blob/master/language/question_answering/bert_joint/run_nq.py#L879-L932) it doesn't seems to be directly possible to load this model in the current library (with simple hacks).

You will need to define a new sub-class of `BertModel` (e.g. `BertForNaturalQA`) that reproduce the architecture of the TensorFlow model I pointed to. If you use the same name as the TensorFlow variables for the attributes of your PyTorch model you should be able to load the model with the current loading script.

I don't have time to do this right now but if you want to start opening a PR, I can review it.

Basically, just add another class after the `BertForQuestionAnswering` class in [`modeling.py`](https://github.com/huggingface/pytorch-pretrained-BERT/blob/master/pytorch_pretrained_bert/modeling.py#L1130)

**Comment by raheja** [comment]
Thanks a lot for the reply. Will try this out.

Best,

**Comment by paulachocron** [comment]
Hi @raheja, do you have any updates related to this? I'm trying to do the same and would welcome any hint!

**Comment by raheja** [comment]
Hi @paulachocron 
Haven't been able to try or implement this yet. I am still using the tf version for now.
